Numsense.UnitTests/DutchExamples.fs

```diff
@@ -0,0 +1,264 @@
+[<InlineData(                                               "zevenenzeventig",         77)>]
+[<InlineData(                                               "negenenzeventig",         79)>]
+[<InlineData(                                                       "tachtig",         80)>]
+[<InlineData(                                                  "zesentachtig",         86)>]
+[<InlineData(                                                      "negentig",         90)>]
+[<InlineData(                                                "drieënnegentig",         93)>]
+[<InlineData(                                                       "honderd",        100)>]
+[<InlineData(                                                    "honderdeen",        101)>]
+[<InlineData(                                                    "honderdéén",        101)>]
+[<InlineData(                                                "honderd en een",        101)>]
+[<InlineData(                                                "honderd en één",        101)>]
+[<InlineData(                                                   "tweehonderd",        200)>]
+[<InlineData(                                      "tweehonderdzevenendertig",        237)>]
+[<InlineData(                                      "driehonderdvijfenzeventig",       375)>]
+[<InlineData(                                           "vierhonderdnegentig",        490)>]
+[<InlineData(                                      "vijfhonderddrieënzestig",         563)>]
+[<InlineData(                                            "zeshonderdachttien",        618)>]
+[<InlineData(                                   "zevenhonderdzevenenzeventig",        777)>]
+[<InlineData(                                               "achthonderdvijf",        805)>]
+[<InlineData(                                            "achthonderd en zes",        806)>]
+[<InlineData(                                            "achthonderdenzeven",        807)>]
+[<InlineData(                                          "achthonderd en zeven",        807)>]
+[<InlineData(                                    "negenhonderdnegenenvijftig",        959)>]
+[<InlineData(                                                       "duizend",       1000)>]
+[<InlineData(                                                   "duizend een",       1001)>]
+[<InlineData(                                                   "duizend één",       1001)>]
+[<InlineData(                                                "duizend en een",       1001)>]
+[<InlineData(                                                "duizend en één",       1001)>]
+[<InlineData(                                          "duizend vierenzestig",       1064)>]
+[<InlineData(                                                 "twaalfhonderd",       1200)>]
+[<InlineData(                                 "zestienhonderdzevenenzeventig",       1677)>]

```

Comment:
I think the formal notation of `duizenddriehonderdzevenendertig` is more common, at least in Belgium. The formal style is the better choice here, imo.